### ISSUE TITLE
[Linux] App exclusions not working on Ubuntu 22 LTS

### ISSUE THREAD

**Issue by blueapple128** [mozilla-mobile/mozilla-vpn-client#9597]
**Describe the bug**
On Ubuntu 22 LTS, app exclusions do not work (the app(s) behave as though no app exclusions were checked).

**VPN version:**
2.22.0

**Affected Platforms:**
Ubuntu 22 LTS

**Steps to reproduce:**
- Boot machine (Mozilla VPN is not set to autostart)
- Open Mozilla VPN (not connected yet) and check Firefox as one of the app exclusions
- Open Firefox and navigate to mullvad.net

**Expected result:**
mullvad.net should state that the user is currently not connected to Mullvad VPN.

**Actual result:**
mullvad.net states that the user is connected to Mullvad VPN.

**Additional notes:**
This happens for any app or combination of apps I've tried (Chromium, Firefox Developer Edition, Terminal->lynx, etc.). I am not sure where to start debugging this.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-6413)

**Comment by blueapple128** [comment]
Resolved; it looks like `sudo apt install mozillavpn` without first adding the `ppa:mozillacorp/mozillavpn` repository tries to install Mozilla VPN anyway, but it installs an older version that's incompatible with app exclusions. If the user then adds the repository, followed by any combination of installing or uninstalling/purging mozillavpn, it seems to ignore this conflicting older version, whose existence seems to prevent [app exclusions in the new installation from the correct repo] from working.

To fix, I had to purge mozillavpn (the current version), then remove the repo with `sudo add-apt-repository -r ppa:mozillacorp/mozillavpn`, then purge mozillavpn again (the old version), then add the repo back. I also heavily restarted the machine a lot in between several of these steps, though I don't remember which exactly or if this would have any effect. Hopefully this helps someone.
